hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -49,6 +49,12 @@
       <artifactId>hadoop-annotations</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop.thirdparty</groupId>
+      <artifactId>hadoop-shaded-guava</artifactId>
+      <scope>compile</scope>
```

Comment:
nit: no need of the compile scope since it default?

Reply:
Thanx, Will update in next iteration. If no further comments. :-)